proto/Networking/Responses/UseItemMoveRerollResponse.proto

```diff
@@ -6,6 +6,7 @@ import "POGOProtos/Data/PokemonData.proto";
 message UseItemMoveRerollResponse {
 	Result result = 1;
 	.POGOProtos.Data.PokemonData upgraded_pokemon = 2;
+	.POGOProtos.Data.PokemonData next_upgraded_pokemon = 3;
```

Comment:
I don't see this

Reply:
Heu! Peut-être sûrement il y a des trucs que j’ai du louper si n’existe plus faudra le supprimer

Reply:
fixed https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/b3e1f463fe2796f0bbd862d55d931ea099991d1f